tests/sqlparser_common.rs

```diff
@@ -1376,13 +1377,314 @@ fn parse_literal_timestamp() {
                 hour: 1,
                 minute: 23,
                 second: 34,
-                nano: 555_000_000
+                nano: 555_000_000,
+                timezone_offset_second: 0,
             }
         )),
         expr_from_projection(only(&select.projection)),
     );
 }
 
+#[test]
+fn parse_literal_timestamptz() {
+    let time_formats = [
+        "TIMESTAMP",
+        "TIMESTAMPTZ",
+        "TIMESTAMP WITH TIME ZONE",
+        "TIMESTAMP WITHOUT TIME ZONE",
+    ];
+
+    let test_cases = [
```

Comment:
you can disable rustfmt with `#[rustfmt::skip]` just above this `test_cases` list, this format seems hard to understand.